data/models/diagnostic.py

```diff
@@ -1686,7 +1686,7 @@ def appro_badge(self):
     @property
     def waste_badge(self):
         if self.has_waste_diagnostic and self.waste_actions and len(self.waste_actions) > 0:
-            if self.canteen.daily_meal_count < 3000 or self.has_donation_agreement:
+            if self.canteen.daily_meal_count and (self.canteen.daily_meal_count < 3000 or self.has_donation_agreement):
```

Comment:
```suggestion
            if self.has_donation_agreement or (self.canteen.daily_meal_count and self.canteen.daily_meal_count < 3000):
```
je ferai plutôt ça, si je pinaille

Reply:
d'ailleurs on ne demande pas un daily_meal_count aux CC sans site